src/main/java/uk/gov/hmcts/reform/sendletter/notify/INotifyClient.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.sendletter.notify;
+
+import uk.gov.hmcts.reform.sendletter.model.Letter;
+
+public interface INotifyClient {
```

Comment:
Shouldn't `NotifyClientStub` implement the same interface as the actual NotifyApiClient?

Reply:
it will, once it's added (separate story)